include/DataFile.h

```diff
@@ -28,9 +28,9 @@
 
 #include <map>
 #include <QDomDocument>
+#include <vector>
```

Comment:
Why was `vector` included here?

Reply:
`vector` was included in `MemoryManager.h`, which was included in `DataFile.h`, but after removing `MemoryManager.h`, `vector` wasn't getting included in `DataFile.h` anymore, so it wouldn't compile.

Reply:
Odd that `DataFile` was even including `MemoryManager.h` to begin with, but okay, this is fine :+1:.